codebuild/codebuild_build_container_ecr/codebuild_build_container_ecr.yml

```diff
@@ -15,11 +15,11 @@ phases:
       - echo Logging into Dockerhub...
       - docker login -u $DOCKERHUB_USERNAME -p $DOCKERHUB_PASSWORD
       - ACTIONS_REQUIRED="${CODEBUILD_SRC_DIR_actions_required}/actions_required.json"
-      - if [ -n "$CHECK_TRIGGER" ]; then
-          TRIGGER_BUILD=$(jq --arg ACTION_NAME "$ACTION_NAME" '.[] | select(.action ==$ACTION_NAME).required' $ACTIONS_REQUIRED); 
-        else 
+      - if [ -z  "$CHECK_TRIGGER" ] || [ "$CHECK_TRIGGER" -eq 0 ]; then
           echo "CHECK_TRIGGER not supplied, defaulting to true..";
-          TRIGGER_BUILD=true; 
+          TRIGGER_BUILD=true;
+        elif [ ! -z "$CHECK_TRIGGER" ] && [ "$CHECK_TRIGGER" -eq 1 ]; then
```

Comment:
See earlier comment (sorry for the spam, just wanted to make sure we didn't miss any of them)